docs/architecture/grpc-for-wcf-developers/introduction.md

```diff
@@ -1,11 +1,63 @@
 ---
 title: Introduction - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: Introduction
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # Introduction
 
+Helping machines communicate with each other has been one of the primary preoccupations of the digital age. In particular, there is an ongoing effort to determine the optimal remote communication mechanism that will suit the interoperability demands of the current infrastructure. As you can imagine, that mechanism changes as either the demands or the infrastructure evolves.
+
+The release of .NET Core 3.0 marks a shift in the way that Microsoft delivers remote communication solutions to developers who want to deliver services across a range of platforms. .NET Core doesn't offer Windows Communication Foundation (WCF) out of the box but, with the release of version ASP.NET Core 3.0, it does provide built-in gRPC functionality.
+
+gRPC is a hugely popular framework in the wider software community, used by developers across many programming languages for modern RPC scenarios. The community and the ecosystem are vibrant and active, with support for the gRPC protocol being added to infrastructure components like Kubernetes, service meshes, load balancers and more. These factors, as well as its performance, efficiency and cross-platform compatibility, make it the natural choice for a successor to WCF.
```

Comment:
```suggestion
gRPC is a popular framework in the wider software community, used by developers across many programming languages for modern RPC scenarios. The community and the ecosystem are vibrant and active, with support for the gRPC protocol being added to infrastructure components like Kubernetes, service meshes, load balancers and more. These factors, as well as its performance, efficiency and cross-platform compatibility, make gRPC a good choice for WCF apps moving to .NET Core.
```

I'm not sure if we should explicitly say gRPC is WCF's successor. That was more the *vibe* I wanted to get across 😄